### ISSUE TITLE
Invalid Degree Plans according to Curricular Analytics

### ISSUE THREAD

**Issue by ArturoAmaya** [ArturoAmaya/ExploratoryCurricularAnalytics#53]
After that thing with BE28 I thought it might be interesting to pre-check the degree plans with the Curricular Analytics Julia package. I took a stab at it, and came up with the following errors:

- BE25: MATH 18 in term 5 is a pre-req for MAE 40 but they're in the same term
- BE28: BENG168 in term 11 is a pre-req for BENG 187A in term 9
- LN34: LIGN 110 in term 7 is a pre-req for 'LIGN 112 or 113' (its a pre-req for both) in term 5
- LT34 (Seventh college only): SYN 2 in term 4 is a pre-req for LTWR 8A or 8B or 8C in the same term
- NA25: PHYS 2D and MATH 20D are both pre-reqs for NANO 102 and occur in the same term or after. (SUS)
- SI30: MATH 18 in term 6 is a pre-req for MATH20E in term 5

I believe, pending a double check, that these are all legitimate errors except for the NA25 one: according to the catalog 2D and 20D are not pre-reqs for NANO 102.

**Comment by ArturoAmaya** [comment]
IK you have a solve for these errors, I just wanted to flag them and bring them up with the respective departments at some point. Also, for NA25 it seems like maybe we were given an outdated list of prereq chains. Just looking at NANO, NANO 101 is also wonky

**Comment by ArturoAmaya** [comment]
I'm going to edit the prereqs document to remove the extra NANO 102 requisites

**Comment by SheepTester** [comment]
> * BE28: BENG168 in term 11 is a pre-req for BENG 187A in term 9

I think the issue here is that unlike the other bioengineering majors, BE28 doesn't have BENG 112A, which is an alternative prereq for BENG 187A, since the other bioengineering majors also have BENG 168 after BENG 187A

**Comment by ArturoAmaya** [comment]
Yeah that's the ticket but no idea why that happens. The one that really bothers me is the 20E/MATH 18 one.
